apps/src/sites/studio/pages/levels/show.js

```diff
@@ -102,6 +104,21 @@ function initPage() {
       'rubric-fab-mount-point'
     );
     if (rubricFabMountPoint) {
+      //rubric fab mount point is only true for teachers
+      if (
+        experiments.isEnabled('ai-rubrics') &&
+        !!rubric &&
+        rubric.learningGoals.some(lg => lg.aiEnabled)
+      ) {
```

Comment:
this looks like a good start, but can you please further limit this so that we only log when the teacher is looking at the final level in the progression, where ai evaluation results are displayed. for example:
    * should trigger on http://localhost-studio.code.org:3000/s/csd3-2023/lessons/11/levels/5?section_id=X&user_id=X
    * should NOT trigger on http://localhost-studio.code.org:3000/s/csd3-2023/lessons/11/levels/4?section_id=X&user_id=X

Reply:
Updated this